integrations/langchain/skyvern_langchain/settings.py

```diff
@@ -0,0 +1,18 @@
+from typing import Literal
+
+from dotenv import load_dotenv
+from pydantic_settings import BaseSettings
+
+
+class Settings(BaseSettings):
+    credential: str = ""
+    base_url: str = "https://api.skyvern.com"
+    engine: Literal["TaskV1", "TaskV2"] = "TaskV2"
+    run_task_timeout: int = 60 * 60
```

Comment:
in the code it's using `run_task_timeout_seconds`. Shall we keep the name consistent?